docs/Artifacts definition format and style guide.asciidoc

```diff
@@ -207,6 +216,7 @@ Currently the following different source types are defined:
 | Value | Description
 | ARTIFACT_GROUP | A source that consists of a group of other artifacts.
 | COMMAND | A source that consists of the output of a command.
+| DIRECTORY | A source that consists of the contents of directories.
```

Comment:
Future of DIRECTORY depends on https://github.com/ForensicArtifacts/artifacts/issues/286

Reply:
But is it currently part of the spec? It is a used in 14 artifact definitions.